pkg/certsuite/certsuite.go

```diff
@@ -48,7 +48,7 @@ func LoadChecksDB(labelsExpr string) {
 
 const (
 	junitXMLOutputFileName = "cnf-certification-tests_junit.xml"
-	collectorAppEndPoint   = "http://44.195.143.94"
+	collectorAppDNS        = "http://claims-collector.cnf-certifications.sysdeseng.com"
```

Comment:
DNS stands for Domain Name Service. Did you mean `collectorAppName`?

Reply:
I meant the DNS, as this URL is using the DNS and not the IP

Reply:
@shirmoran DNS is a service that is run by a server. Perhaps you meant URL?

Reply:
Yeah, `collectorAppUrl` is a better name.